app/controllers/api/mixins/service_templates.rb

```diff
@@ -14,6 +15,11 @@ def order_service_template(id, data, scheduled_time = nil)
 
       private
 
+      def request_from_ui?
+        return false if request.headers["x-auth-token"].blank?
+        !token_info.empty?
```

Comment:
You'll need the definition of `token_info` from https://github.com/ManageIQ/manageiq-api/pull/476/files#diff-3db7e02fbc903162f12ccf71cbc6bab6R33 as well.